src/components/PermissionPanel/index.tsx

```diff
@@ -0,0 +1,273 @@
+import { useQuery } from "@apollo/client";
+import { ArrowDropDown } from "@mui/icons-material";
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Checkbox,
+  FormControlLabel,
+  FormGroup,
+  styled,
+  Typography,
+} from "@mui/material";
+import { FC, memo, useEffect, useMemo, useRef } from "react";
+import { useFormContext } from "react-hook-form";
+import Grid2 from "@mui/material/Unstable_Grid2";
+import { useSnackbar } from "notistack";
+import { cloneDeep } from "lodash";
+import {
+  EditUserInput,
+  RetrievePBACDefaultsResp,
+  RetrievePBACDefaultsInput,
+  RETRIEVE_PBAC_DEFAULTS,
+} from "../../graphql";
+import { ColumnizedPBACGroups, columnizePBACGroups, Logger } from "../../utils";
+
+const StyledBox = styled(Box)({
+  width: "957px",
+  transform: "translateX(-50%)",
+  marginLeft: "50%",
+  marginTop: "63px",
+});
+
+const StyledAccordion = styled(Accordion)({
+  width: "100%",
+});
+
+const StyledAccordionSummary = styled(AccordionSummary)({
+  borderBottom: "1px solid #6B7294",
+  minHeight: "unset !important",
+  "& .MuiAccordionSummary-content": {
+    margin: "9px 0",
+  },
+  "& .MuiAccordionSummary-content.Mui-expanded": {
+    margin: "9px 0",
+  },
+  "& .MuiAccordionSummary-expandIcon": {
```

Comment:
This is not correctly targeting the expand icon, I still see it as gray. I believe it should be `.MuiAccordionSummary-expandIconWrapper` instead.

Reply:
Good catch. Updated in 4553245.